File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    var answer = [];
    let loc_y = []
    let loc_x = []
    
    for(var i=0; i<wallpaper.length; i++){
        for(var j=0; j<wallpaper[i].length; j++){
            if(wallpaper[i][j] === '#'){
                loc_y.push(i)
                loc_x.push(j)
            } 
        }
    }
    // 최솟값
    answer.push(Math.min(...loc_y))
    answer.push(Math.min(...loc_x))
    //최댓값
    answer.push(Math.max(...loc_y)+1)
    answer.push(Math.max(...loc_x)+1)
    
    return answer;
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    answer = []
    # x따로 y따로
    loc_x = []
    loc_y = []
    
    for y_idx,row in enumerate(wallpaper):
        for x_idx,target in enumerate(row):
            if target == '#':
                loc_y.append(y_idx)
                loc_x.append(x_idx)
    
    answer = [min(loc_y),min(loc_x),max(loc_y)+1,max(loc_x)+1]
    
    return answer